tests/test_schema.py

```diff
@@ -200,15 +200,18 @@ def test_normalize_float():
     assert normalize_field_type("float:25.8") == "float"
 
 
+def test_normalize_():
+    assert normalize_field_type("float:25.8") == "float"
+
+
 def generate_field_types():
     """
     Produce a unique set of field types in a consistent order.
 
     This ensures that tests are able to run in parallel.
     """
-    types = set(FIELD_TYPES)
-    types.remove(None)
-    return list(sorted(types)) + [None]
+    types = set(NAMED_FIELD_TYPES.keys())
```

Comment:
`FIELD_TYPES` remains, but internal usage now uses the more complete `NAMED_FIELD_TYPES` to get, for example `FionaBooleanType` by its name `"bool"`.